packages/repository-tests/package.json

```diff
@@ -13,7 +13,7 @@
     "build": "lb-tsc",
     "clean": "lb-clean loopback-repository-tests*.tgz dist tsconfig.build.tsbuildinfo package",
     "pretest": "npm run build",
-    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\" --reporter=spec",
```

Comment:
Please revert this change. You can temporarily enable `spec` reporter as follows:

```
$ npm test -- --reporter=spec
```